### PR TITLE
feat(APM logs in context): screenshots

### PR DESCRIPTION
**Pull request by barbnewrelic**
For DOC-7769. Approved by SMEs but needs full TW peer edit: structure, use of images, copyedits, etc. Thanks!

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Review by paperclypse — APPROVED** [review]
I think this looks good! I left some codestream comments that you can feel free to ignore in the short term. Nothing that merits holding off on merging this in.

**Comment by barbnewrelic** [comment]
Incorporated peer review feedback and discussed with my editor. This does not need another review.

**Pull request merged**
